src/UI/MainWindow.cpp

```diff
@@ -43,17 +53,22 @@ MainWindow::MainWindow(const std::function<void(const std::string&)>& radioUploa
   downloadAct->setStatusTip(tr("Download file"));
   fileMenu->addAction(downloadAct);
 
+  QAction* exportAct = new QAction(tr("Export to ConfFile"), this);
```

Comment:
Jesus christ. Probably should move some options around. Upload/download to a Radio menu or something.